app/moves/middleware.js

```diff
@@ -95,19 +62,20 @@ module.exports = {
     next()
   },
   setPagination: (req, res, next) => {
-    const { moveDate, period = 'week', status = '' } = res.locals
+    const { period = 'week' } = res.locals
+    const { status = '' } = req.params
```

Comment:
Could also get `period` from `req.params` for consistency. That way, the middleware order isn't as important.

Reply:
I don't think it needs a default value either to be honest. If it doesn't exist it will be `undefined` which is enough for the falsy checks you currently have.

Reply:
Also, you're destructing the req.paras object on line 68 so you can combine those.